providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/FlagdOptionsTest.java

```diff
@@ -31,6 +33,7 @@ public void TestDefaults() {
         assertNull(builder.getSelector());
         assertNull(builder.getOpenTelemetry());
         assertNull(builder.getOfflineFlagSourcePath());
+        assertEquals(Config.Evaluator.RPC, builder.getResolverType());
```

Comment:
I was just looking at this, and I was wondering if it was a good idea to change `Evaluator` here to `Resolver` for consistency. It would be breaking, but simple. What do you guys think @beeme1mr @Kavindu-Dodan ?

Reply:
Good point, we can do a deprecation and introduce a new replacement so it won't be a breaking change

Reply:
I think that's good.

Reply:
I will create a follow-up and let's add these changes to the next release

Reply:
Opened draft PR https://github.com/open-feature/java-sdk-contrib/pull/793 

Commit https://github.com/open-feature/java-sdk-contrib/pull/793/commits/60665a35ba7d0cb6513fa5307e8d946ca7c0b6b5 summarize the change :)